pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-patron</artifactId>
-  <version>5.1.0-SNAPSHOT</version>
+  <version>5.1.1-SNAPSHOT</version>
```

Comment:
I believe this is a mainline feature release.

The next version for a mainline release should be the next minor or major (as described [here](https://dev.folio.org/guidelines/release-procedures/#prepare-and-perform-the-source-release)), not a patch version. As I understand it, patch versions should only be used in the long lived release branches.

This version should be aligned with the version in the news (which is a minor increment).

```suggestion
  <version>5.2.0-SNAPSHOT</version>
```

Reply:
updated